libraries/Wire/src/Wire.cpp

```diff
@@ -66,9 +66,11 @@ void TwoWire::begin(uint8_t address)
 {
   rxBufferIndex = 0;
   rxBufferLength = 0;
+  rxBuffer = resetBuffer(rxBuffer);
```

Comment:
if null raise en error ?

Reply:
Error is raised later if rx/txBuffer is NULL